packages/@aws-cdk/aws-dynamodb/lib/table.ts

```diff
@@ -1169,6 +1175,7 @@ export class Table extends TableBase {
       },
       sseSpecification,
       streamSpecification,
+      tableClass: props.tableClass ? props.tableClass : undefined,
```

Comment:
No reason for this dance 🙂. Just this is fine:

```suggestion
      tableClass: props.tableClass,
```